Framework/DataHandling/src/LoadEMU.cpp

```diff
@@ -687,9 +759,21 @@ template <typename FD> void LoadEMU<FD>::exec(const std::string &hdfFile, const
   bool saveAsTOF = !Base::getProperty(RawDopplerTimeStr);
   bool loadAsTOF = !m_calibrateDoppler && saveAsTOF;
   EMU::EventAssigner eventAssigner(roi, detMapIndex, PIXELS_PER_TUBE, framePeriod, gatePeriod, timeBoundary,
-                                   directLimits, analysedLimits, convertTOF, eventVectors, start_nanosec, loadAsTOF);
+                                   directLimits, analysedLimits, convertTOF, eventVectors, start_nanosec, loadAsTOF,
+                                   includeBM);
   EMU::loadEvents(prog, "loading neutron events (TOF)", eventFile, eventAssigner);
 
+  // determine the minimum and maximum beam rate per sec
+  auto filteredBM = runningAverage(eventAssigner.beamMonitorCounts(), 5);
```

Comment:
It's not clear to me why the hard-coded value `5` is used here - is it worth adding a comment why that value is chosen?

Reply:
Added an explanation for the choice of the half window size.